docs/source/contributors/guidelines.rst

```diff
@@ -19,9 +19,8 @@ The workflow to be used for submitting new code/issues is described in
 Linting
 =======
 
-All pull requests should be `PEP 8 compliant <https://www.python.org/dev/peps/pep-0008/>`_.
-We suggest running code through `flake8 <https://flake8.pycqa.org/en/latest/>`_ and
-`pylint <https://www.pylint.org/>`_ before submitting to check for this.
+All pull requests should be compliant with selected `Ruff<https://docs.astral.sh/ruff/>`_ rules.
```

Comment:
Not yet.I have not added that ruleset. I created an issue for this #1329. It will make the variable naming compliant. But there will need to be about a 100 changes. Our repo is not compliant yet. So, I just removed that because we were not checking for it explicitly.